### PR TITLE
Pocketbook: Initial PB633 color support

### PR DESCRIPTION
**Pull request by ezdiy**
Depends on https://github.com/koreader/koreader-base/pull/1191

Needs feedback https://github.com/koreader/koreader/issues/6479

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6680)

### CODE REVIEW

File: frontend/device/pocketbook/device.lua [hunk 1]
```diff
@@ -21,6 +21,10 @@ local ext_path = "/mnt/ext1/system/config/extensions.cfg"
 local app_name = "koreader.app"
 
 local PocketBook = Generic:new{
+    canUseCBB = no,
+    hasColorScreen = yes,
```
Comment (poire-z) [inline frontend/device/pocketbook/device.lua:25]: You probably meant `= no` (generic PB has no color screen), to override it with `yes` for the color devices.
Same for hasCFAScreen = no just below.

Comment (ezdiy) [reply frontend/device/pocketbook/device.lua:25]: Temporary brute force to make CFA masking shown on all, even B&W devices (ie mine).

**Comment by pazos** [comment]
Ping @Teteros @axispod

**Comment by pazos** [comment]
Superseeded by https://github.com/koreader/koreader/pull/8727

**Pull request closed without merging**